packages/badge/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { Products } from '@hashicorp/platform-product-meta'
+import classnames from 'classnames'
+import s from './style.module.css'
+
+interface BadgeProps {
+  children: string
+  theme?: Products | 'neutral' | 'white' | 'black'
+  variant?: 'primary' | 'secondary'
+}
+
+export default function Badge({
+  children,
+  variant = 'primary',
+  theme = 'neutral',
+}: BadgeProps) {
+  return (
+    <span className={classnames(s.badge, s[theme], s[variant])}>
```

Comment:
I'll defer to @ashleemboyer, but I vaguely recall badges like this benefiting from some aria props like `role="status"`. That pertains more to badges that are being updated as a result of user action which doesn't seem to be the intent here, but I'm curious if we should go ahead and add those attributes now to support that use case in the future.

Reply:
Setting `role="status"` has side effects, like setting `aria-live` to `polite`, so I'd say let's hold off on adding ARIA until we know more about how this component will be used. But my gut take is that if the text in this badge is static / doesn't update while a user is looking at it, then `role="status"` shouldn't be used.

Source: https://www.w3.org/TR/wai-aria/#status